src/cpu/core_dyn_x86/cache.h

```diff
@@ -479,19 +487,22 @@ static INLINE void cache_addb(Bit8u val) {
 	*cache.pos++=val;
 }
 
-static INLINE void cache_addw(Bit16u val) {
-	*(Bit16u*)cache.pos=val;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).